### DISCUSSION TITLE
DNSecure formula for `brew`?

### DISCUSSION THREAD

**Discussion by fooness** [kkebo/DNSecure#153] [Ideas]
Hello everyone; thank you for this superb application for iOS and macOS.

I wondered, is there any interest in maybe making available `DNSecure` not only via App Store, but also via [`brew`](https://docs.brew.sh/Adding-Software-to-Homebrew#testing-your-new-cask)?

Thank you very much in advance.

**Comment by kkebo** [comment]
Thank you for your suggestion.

I don't have any plans to release DNSecure in Homebrew at this time, but if there is a good source of information about publishing a Mac Catalyst app in Homebrew, I will try to start working on it.

**Comment by Aaron-Rumpler** [comment]
For applications like this, someone would write a Homebrew cask, which would simply download the application (the `.app`, normally in a DMG) from wherever you host it and copy it to `/Applications`. So all that would be required on your part is to make DNSecure available for download outside of the Mac App Store if you wanted it to be available through Homebrew.
